tests/unit/amo/pages/TestAddon.js

```diff
@@ -507,6 +508,7 @@ describe(__filename, () => {
     const clientApp = CLIENT_APP_FIREFOX;
     const { store } = dispatchClientMetadata({ clientApp });
     const addon = fakeAddon;
+    const addonURL = getAddonURL(addon.slug);
```

Comment:
In all the tests in this file, there's no need to define a `const` for `addonURL`. The call to `getAddonURL(addon.slug)` can just me made inline.